packages/driver/cypress/e2e/commands/misc.cy.js

```diff
@@ -160,15 +160,6 @@ describe('src/cy/commands/misc', () => {
       })
     })
 
-    // TODO: fix this test in 4.0 when we refactor validating subjects
```

Comment:
This test has been skipped for ages; I investigated it, attempted a fix, and discovered that it's actually incorrect - we don't want to validate that jQuery collections contain only DOM nodes, because text nodes (strings) are also perfectly valid members of a collection.

In short - old test, skipped forever, Cypress has moved on.